operator/apis/platform/v1alpha1/common_types.go

```diff
@@ -146,6 +146,9 @@ type ScannerV4Component struct {
 
 // ScannerV4DB defines configuration for the Scanner V4 database component.
 type ScannerV4DB struct {
+	// Configures how Scanner V4 should store its persistent data.
+	// You can choose between using a persistent volume claim (recommended default), a host path,
+	// and an emptyDir volume in case Scanner V4 is running on a secured cluster.
```

Comment:
```suggestion
	// or an emptyDir volume if Scanner v4 is running on a secured cluster.
```

Reply:
I think it was aligned on "Scanner V4" as the official way of referring to this thing -- with a capital "V".

Reply:
I've changed everything I found to be "Scanner V4"